src/Coff.zig

```diff
@@ -73,6 +73,23 @@ pub fn flush(self: *Coff) !void {
     }
 
     try self.parsePositionals(positionals.items);
+    for (self.objects.items) |*obj| {
+        log.debug("Object name: '{s}'", .{obj.name});
+        log.debug("Symbols: {}", .{obj.symtab.len()});
+        var it = obj.symtab.slice(0, null);
+        while (it.next()) |sym| {
+            const name = sym.getName() orelse obj.getString(sym.getNameOffset().?);
+            log.debug("    Name: `{s}`", .{name});
+            log.debug("        Type:    {} {}", .{ sym.type.base_type, sym.type.complex_type });
+            log.debug("        Storage: {}", .{sym.storage_class});
+        }
+        log.debug("SectionHeaders: {}", .{obj.shdrtab.items.len});
+        for (obj.shdrtab.items) |sh| {
+            const name = sh.getName() orelse obj.getString(sh.getNameOffset().?);
+            log.debug("    Name: `{s}`", .{name});
+            log.debug("    Flags: {x}", .{@as(u32, @bitCast(sh.flags))});
```

Comment:
Instead of having `log.debug` scattered for `Object` in `Coff`, how about we approach the same way I did with `Elf`, i.e., having a custom formatter(s) in `Object` and then calling `log.debug("{}", object.fmt(self))` or similar?